templates/actors/tabs/character-spells.hbs

```diff
@@ -120,6 +128,16 @@
                         </div>
                     </div>
 
+                    {{!-- Spell Uses --}}
+                    <div class="item-detail item-uses spell-uses {{#unless ctx.hasUses}}empty{{/unless}}">
+                        {{#if ctx.hasUses}}
+                        <input type="text" value="{{ item.system.uses.value }}" placeholder="0" data-dtype="Number"
+                               data-name="system.uses.value" inputmode="numeric" pattern="[0-9+=\-]*">
```

Comment:
I made a mistake with these delta expressions and have been trying to slowly fix them up wherever I find them

```suggestion
                               data-name="system.uses.value" inputmode="numeric" pattern="[+=\-]?\d*">
```